### PR TITLE
feat: feat/REPLAT_1212 Pull Quote Design Update

### PR DESCRIPTION
**Pull request by jimhunty**
After Pull Quotes was design reviewed, Jake further refined the design. As a result, this now includes a twitter icon.

As the twitter icon is also used in the author head, it made sense to centralise an icons package, which is included in this PR.

This is how the Pull Quotes look now:

### Android
screen shot 2018-01-12 at 17 03 25

### iOS
screen shot 2018-01-12 at 17 03 06

### Mobile Web
screen shot 2018-01-12 at 17 02 38

### Tablet
screen shot 2018-01-12 at 17 02 49

### Desktop
screen shot 2018-01-12 at 17 02 56

### CODE REVIEW

File: packages/icons/__tests__/android/icons.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+jest.mock("react-native", () => {
+  const reactNative = require.requireActual("react-native");
+  reactNative.Platform.OS = "android";
+  jest
+    .spyOn(reactNative.Platform, "select")
+    .mockImplementation(obj => obj.android || obj.default);
+  return reactNative;
+});
+
+jest.mock("WebView", () => "WebView");
```
Comment (craigbilner) [inline packages/icons/__tests__/android/icons.test.js]: not needed slash this config also needs centralising at some point

Comment (jimhunty) [reply packages/icons/__tests__/android/icons.test.js]: Removed. This comes from the CLI which I have now updated. In this respect it is already centralised, but probably better as a package variable.

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/cbf24b3ae7d7159e

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/7b8064f0f7b56307

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/846dbc84df971a06

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://www.bitrise.io/build/55ee50ca3e1c8d8e

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/$BITRISE_GIT_COMMIT/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/something/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/064aa1d72739641edade0f63dc1c8c999128736f/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/363a0f930a0ba01d89a6b0632ff4a3e04b1523a0/dextrose-test/index.html

**Pull request merged**